tests/Aspire.Playground.Tests/Aspire.Playground.Tests.csproj

```diff
@@ -17,11 +17,17 @@
     <PlaygroundSourceDir>$(MSBuildThisFileDirectory)..\..\playground\</PlaygroundSourceDir>
     <TestsSharedDir>$(MSBuildThisFileDirectory)..\Shared\</TestsSharedDir>
     <RunSettingsFilePath>$(MSBuildThisFileDirectory).runsettings</RunSettingsFilePath>
+    <!-- Skip emulators that don't start consistently when running in CI. -->
+    <SkipUnstableEmulators Condition="'$(SkipUnstableEmulators)' == '' and ('$(RepoRoot)' == '' or '$(ContinuousIntegrationBuild)' == 'true' or '$(CODESPACES)' == 'true')">true</SkipUnstableEmulators>
 
     <!-- on helix this will be available in the source dir -->
     <XunitRunnerJson Condition="'$(RepoRoot)' == ''">xunit.runner.json</XunitRunnerJson>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(SkipUnstableEmulators)' == 'true'">
```

Comment:
NIT: Can this just be done in the parent's Directory.Build.props so it applies to both playground and tests and that way we don't have to change multiple places later?

Feel free to not block on this one, but just something to consider.

Reply:
Note that we already had this code in the repo.

* Orignally added in https://github.com/dotnet/aspire/pull/5418
* Renamed in https://github.com/dotnet/aspire/pull/5827
* Removed in https://github.com/dotnet/aspire/pull/6737
* and now brought back with another different name.

One reason to not put it in the root of the repo is so it doesn't affect other projects - like stuff under `src`.